vanilla-framework/vanilla-framework
# Update docs/lists page content

## Done
- Updated copy ✏️ 
- Updated demo examples 💯 
- Renamed components to match copy document
- Check content matches copy doc [here](https://docs.google.com/document/d/1ZFRd4z7TsU4oifxgky77337v_1fkE2ak12tGPNivD70/edit#heading=h.ifh1f0ulklnm)

## QA
- Pull code
- Run /docs
  - Check http://0.0.0.0:8104/en/patterns/lists
- Pull code
- Run ./run
- Review many many examples 😉 
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/list/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-dividers/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-stepped-detailed/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-inline/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-mid-dot/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-split/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-dividers-ticked/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-ticked/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/divider/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-stepped/

## Fixes
- Fixes https://github.com/ubuntudesign/design-vanilla-framework/issues/387

## Screenshot
![Screenshot 2019-04-08 at 16 45 36](https://user-images.githubusercontent.com/17748020/55737810-cc522e00-5a1d-11e9-9358-bd0c0144dcd4.png)
